qubes-rpc-policy/80-whonix.policy

```diff
@@ -0,0 +1,13 @@
+# service            arg source            target            action params
+
+whonix.SdwdateStatus  *  @tag:anon-gateway @tag:anon-vm      allow autostart=no
+whonix.SdwdateStatus  *  sys-whonix        @tag:anon-vm      allow autostart=no
+whonix.SdwdateStatus  *  @anyvm            @anyvm            deny
+
+whonix.NewStatus      *  @tag:anon-vm      @tag:anon-gateway allow autostart=no
+whonix.NewStatus      *  @tag:anon-vm      sys-whonix        allow autostart=no
```

Comment:
Should this be allowed with all arguments or only with an empty argument?

Reply:
Good question. Thank you for looking into this! Looking into this now.

Reply:
Just reminded myself by updating https://phabricator.whonix.org/T930 how that works and tested it.

And indeed, "no argument is required" (as in no parameter required). In other words, `sys-whonix` only needs and does only run for example `qrexec-client-vm anon-whonix whonix.SdwdateStatus`.

I'd test this / add a future pull request. Separate ticket for that?